client/src/parser.js

```diff
@@ -0,0 +1,99 @@
+const { parse } = require('path');
+
+/**
+ * @description Parses an JSON created from the .bpmn xml of the model to an executable .robot file
+ * @returns {string} Code that has to be put in .robot file
+ */
+
+// this should be refactored to a dictionary
+function taskLib(app) {
+  if (app == 'Excel.Files') return 'RPA.Excel.Files';
+  if (app == 'Browser') return 'RPA.Browser';
+}
+
+function parseDiagramJson(json_data) {
```

Comment:
This looks like a very long function to me. Can we divide this up? Maybe the sections below each comment correspond to a useful function section?